modelarmor/api/ModelArmor.Samples/ModelArmor.Samples.csproj

```diff
@@ -4,6 +4,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Google.Cloud.Dlp.V2" Version="4.17.0" />
```

Comment:
Or maybe you don't need the dependency on the samples projects, I think you are just using the DLP client library on tests.